bpf/tests/pid_match_test.c

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+/* Copyright Authors of Cilium */
+
+//go:build ignore
+
+#include "vmlinux.h"
+#include "compiler.h"
+#include "bpf_core_read.h"
+#include "bpf_helpers.h"
+#include "process/retprobe_map.h"
+
+#include "process/types/basic.h"
+#include "process/generic_calls.h"
```

Comment:
I dont think this include is needed and AFAICS will cause compilation fail now

Reply:
this is due to pfilter.h, it rely on
-  `struct execve_map_value` declare in lib/process.h
-  `op_filter_in` process/types/operations.h
- `INDEX_MASK` in `process/types/basic.h`

to make in compile, i add this headers. 

Any ideas to fix this?

Reply:
this does it for me

```
diff --git a/bpf/tests/pid_match_test.c b/bpf/tests/pid_match_test.c
index 1411d4a5351f..0af78469f7d8 100644
--- a/bpf/tests/pid_match_test.c
+++ b/bpf/tests/pid_match_test.c
@@ -10,7 +10,6 @@
 #include "process/retprobe_map.h"
 
 #include "process/types/basic.h"
-#include "process/generic_calls.h"
 #include "process/pfilter.h"
 
 char _license[] __attribute__((section("license"), used)) = "Dual BSD/GPL";

```